internal/command/fn/new.go

```diff
@@ -26,10 +26,30 @@ import (
 )
 
 type New struct {
-	Name        string `arg:"" help:"the name of the function"`
-	Language    string `name:"lang" short:"l" required:"" enum:"rust, js" help:"the language of the function"`
-	TemplateDir string `short:"t" type:"path" default:"." help:"the directory where the template are located"`
-	OutDir      string `short:"o" type:"path" default:"." help:"the directory where the function will be created"`
+	Name        string `arg:"" help:"The name of the function"`
+	Language    string `name:"lang" short:"l" required:"" enum:"rust, js" help:"The language of the function"`
+	TemplateDir string `short:"t" type:"path" default:"." help:"The directory where the template are located"`
+	OutDir      string `short:"o" type:"path" default:"." help:"The directory where the function will be created"`
+}
+
+func (n *New) Help() string {
+	return `
+DESCRIPTION
+
+	It creates a new function with specified name from a template.
+	It must be use the flag "--lang" to specify the language of the function.
```

Comment:
As for the build command change the --lang sentence in: The "--lang" flag is required, with the following possible values: [rust, js]

Also change "function destination directory" into "output directory".